pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/store/TextIndexUtils.java

```diff
@@ -300,6 +300,12 @@ public static Constructor<QueryParserBase> getQueryParserWithStringAndAnalyzerTy
     return (Constructor<QueryParserBase>) queryParserClass.getConstructor(String.class, Analyzer.class);
   }
 
+  /**
+   * Writes the config to the properties file.
```

Comment:
Can we be more specific on the fields which are written to the properties files?